### DISCUSSION TITLE
Any plans to support additional languages?

### DISCUSSION THREAD

**Discussion by daveydee33** [speechly/speechly#139] [General]
I'm very interested in using Speechly for my multi language app.  I'm currently using the Speechly polyfill with [react-speech-recognition](https://github.com/JamesBrill/react-speech-recognition).

I'd be interested to know if there are any plans to support additional languages, such as Spanish, Portuguese, Indonesian, Chinese, etc?

Thanks, and looking forward to much success with Speechly! 🤩

**Comment by aukkonen** [comment]
Hi @daveydee33 !

Thank you! :-)

We currently support English and Finnish (as we are based in Helsinki), but English is at the moment our main focus. Spanish support is on the roadmap. For enterprise customers we have the capability to add new languages with a few months lead time. Our speech-recognition technology is built fully in-house, and hence adding support for new languages takes an effort that we must consider on a case-by-case basis.

**Reply by daveydee33** [reply]
Thanks @aukkonen , and I'm looking forward to see the future of speech/audio tech and good luck with Y-Combinator to you and the team! 👏 🎉

**Comment by DeanWronowski** [comment]
Is there a roadmap with rough dates as to when this might happen? :) Great work by the way

**Reply by bigdatabaracus** [reply]
Hello @DeanWronowski, and sorry for the not answering earlier. Your question must have slipped through the cracks. We are looking at releasing Spanish beginning of next year.

**Reply by bigdatabaracus** [reply]
...and French and German to follow by the end of H1/2023.

**Reply by kla-ko** [reply]
Hello @bigdatabaracus ,
Is there an update on the support of other languages?

Thanks
